src/VisualStudio/Core/Def/Implementation/ProjectSystem/CPS/IWorkspaceProjectContext.cs

```diff
@@ -52,6 +54,11 @@ internal interface IWorkspaceProjectContext : IDisposable
         /// </summary>
         void RemoveAnalyzerConfigFile(string filePath);
 
+        /// <summary>
+        /// Renames a file (like a .editorconfig) used to configure analyzers.
+        /// </summary>
+        void RenameAnalyzerConfigFile(string originalFilePath, string newFilePath);
```

Comment:
When will this happen?

Reply:
Move and rename are the same as far as full path is concerned, so if an editorconfig is moved then this should be invoked

Reply:
Ah okay. Right now this will never happen due to the way we find these during design-time build , but when we move this evaluation time we will recognize renames when you use Solution Explorer to move them.